lib/components/contribute_photo_dialog.dart

```diff
@@ -31,6 +31,7 @@ class _ContributePhotoDialogState extends State<ContributePhotoDialog> {
   File _image;
   String _uploadToken;
   bool _isUploading = false;
+  ImagePicker imagePicker = ImagePicker();
```

Comment:
```suggestion
  final _imagePicker = ImagePicker();
```
Per the README for the image_picker package, it looks like this should be final.